src/Core/Resolvers/DWSqlQueryBuilder.cs

```diff
@@ -123,11 +123,11 @@ private static string GenerateColumnsAsJson(SqlQueryStructure structure, bool su
                 // the below ensures there is a comma after id but not after name.
                 if (i != structure.Columns.Count)
                 {
-                    stringAgg.Append(",");
+                    stringAgg.Append("+\',\'+");
```

Comment:
so this is used to escape the single quotes used when appending the char `,` ? can you add more details about the escape sequence usage/ requirement?

Reply:
This just adds the escape characters for the , that we are adding to the string. We are adding a string character , and hence need to literals.